### ISSUE TITLE
[GE Enhancement] Need alt tags for all icons

### ISSUE THREAD

**Issue by PatrickRyanMS** [BabylonJS/Babylon.js#10745]
We need to generate alt tags for all of our icons so that we cover accessibility requirements for the site. Anything that a screen reader cannot read needs an alt tag that is descriptive for the screen reader to be compliant. I believe the react components for things like checkboxes should already be handled, but we should check to be sure.
